packages/avatars/src/index.ts

```diff
@@ -5,5 +5,5 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-export { default as Avatar } from './elements/Avatar';
```

Comment:
no more mix of default vs named exports

Reply:
I think [`MonthSelector` is the only missing default export](https://github.com/zendeskgarden/react-components/blob/c14ebc92c3e97b914019147440a9c3a3b79eb77c/packages/datepickers/src/elements/Datepicker/components/MonthSelector.tsx#L66) for components.

Reply:
As `MonthSelector` was neither an exported component or subcomponent, I didn't have it in view for this PR. But I'll go ahead and update the outlier.